connect/runtime/src/test/java/org/apache/kafka/connect/runtime/rest/resources/ConnectorsResourceTest.java

```diff
@@ -65,8 +67,15 @@
     // URL construction properly, avoiding //, which will mess up routing in the REST server
     private static final String LEADER_URL = "http://leader:8083/";
     private static final String CONNECTOR_NAME = "test";
+    private static final String CONNECTOR_NAME_SPECIAL_CHARS = "test?a=b&c=d\rx=1.1\n><";
```

Comment:
Can you add a few more characters, like space, tab, `%`, `+`, etc.? Some of those are used in the URL encoding, so including them would help ensure we're properly encoding and decoding.